lib/jxl/dec_frame.cc

```diff
@@ -855,7 +855,7 @@ Status FrameDecoder::ProcessSections(const SectionInfo* sections, size_t num,
   if (has_error) return JXL_FAILURE("Error in DC group");
 
   if (*std::min_element(decoded_dc_groups_.begin(), decoded_dc_groups_.end()) ==
-          true &&
+          1 &&
```

Comment:
can also check without ==, probably clearer

Reply:
Yes, that does look nicer. Done.